### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by jennjenn21** [community/community#107631] [Other Feature Feedback, Questions, & Ideas]
Have I been hacked 10 m months or some experimenting on me but most of all is my guthub payment info getting stolen right now

**Comment by jennjenn21** [comment]
> Have I been hacked 10 + months or some experimenting on me but most of all is my guthub payment info getting stolen right now

**Comment by queenofcorgis** [comment]
Hi @jennjenn21 ! The best place to go for answers to account-specific assistance like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
